atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolIntegrationTest.java

```diff
@@ -16,6 +16,7 @@
 package com.palantir.atlasdb.keyvalue.cassandra;
 
 import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertEquals;
```

Comment:
please use AssertJ's assertions rather than junit ones, even where there is precedence in a class (and maybe convert classes you touch). It produces better error messages and is easier to use.